src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -0,0 +1,61 @@
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import { useState } from 'react';
+import StepOne from '../../../src/components/RegisterForm/StepOne';
+import StepTwo from '../../../src/components/RegisterForm/StepTwo';
+
+function RegisterForm() {
+  const [data, setData] = useState({
+    email: '',
+    companyName: '',
+    companyLogin: '',
+    companyPassword: '',
```

Comment:
Yup allows to create a `conditional validationScheme`, thanks to that you could set here the `validationScheme` proper for certain use case which would allow to eliminate passing even empty `password` or `company password` if this case scenario doesn't include it